### PR TITLE
Makefile: fix make clean target when WINEPREFIX directory doesn't exist

### PR DESCRIPTION
**Pull request by tehnick**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -620,7 +620,7 @@ BUILD_PKG_TMP_FILES := *-*.list mxe-*.tar.xz mxe-*.deb* wheezy jessie
 
 .PHONY: clean
 clean:
-	-chmod 0755 "$$WINEPREFIX"
+	@[ -d "$$WINEPREFIX" ] && -chmod 0755 "$$WINEPREFIX" || true
```
Comment (TimothyGu) [inline Makefile]: You could eliminate the `|| true` by using `@-[ -d` IIUC.

**Comment by starius** [comment]
We are going to create a readonly directory in tmp-`<pkg>`-`<target>` before a package build and remove it after a package build.

See https://github.com/mxe/mxe/issues/1221#issuecomment-185110607

File: Makefile (hunk 1 above)
Comment (tehnick) [inline Makefile]: @TimothyGu I have tried:
```
	@-[ -d "$$WINEPREFIX" ] && -chmod 0755 "$$WINEPREFIX"
```
The result is:
```
$ LC_ALL=C make clean
Makefile:623: recipe for target 'clean' failed
make: [clean] Error 1 (ignored)
rm -rf /opt/mxe/tmp-* /opt/mxe/usr \
       ./*-*.list ./mxe-*.tar.xz ./mxe-*.deb* ./wheezy ./jessie
```

**Comment by tehnick** [comment]
@starius This is in your future plans, and I do not know when are you going to do it. Currently I see such output:
```
$ LC_ALL=C LANG=C make clean
chmod 0755 "$WINEPREFIX"
chmod: cannot access '/opt/mxe/usr/readonly': No such file or directory
Makefile:623: recipe for target 'clean' failed
make: [clean] Error 1 (ignored)
rm -rf /opt/mxe/tmp-* /opt/mxe/usr \
       ./*-*.list ./mxe-*.tar.xz ./mxe-*.deb* ./wheezy ./jessie
```
Of course it is up to you to accept or discard proposed simple fix.

**Comment by starius** [comment]
By the way, `make clean` works successfully for MXE `master` branch. The warning "chmod: cannot access '/opt/mxe/usr/readonly': No such file or directory" is ignored by `make` because of `-` before the command, so the exit status is 0.

```
$ make clean
chmod 0755 "$WINEPREFIX"
chmod: cannot access `/tmp/mxe/usr/readonly': No such file or directory
make: [clean] Error 1 (ignored)
rm -rf /tmp/mxe/tmp-* /tmp/mxe/usr \
               ./*-*.list ./mxe-*.tar.xz ./mxe-*.deb* ./wheezy ./jessie

$ echo $?
0
```

Nevertheless, it worth fixing this warning. Regarding the choice between `-` and `|| true`, I prefer using the same style for all parts of this command. You use boolean expression (`[ -d "$$WINEPREFIX" ] &&`) to check for existence, so you should use boolean expression (`|| true`) to mute it instead of Makefile feature (`-`). I propose the following code:

```
[ -d "$$WINEPREFIX" ] && chmod 0755 "$$WINEPREFIX" || true
```

It works:

```
$ make clean
[ -d "$WINEPREFIX" ] && chmod 0755 "$WINEPREFIX" || true
rm -rf /tmp/mxe/tmp-* /tmp/mxe/usr \
               ./*-*.list ./mxe-*.tar.xz ./mxe-*.deb* ./wheezy ./jessie
```

**Comment by tehnick** [comment]
@starius I have updated commit.

**Comment by starius** [comment]
Thanks!

Please rebase your branch:

```
$ git pull --rebase https://github.com/mxe/mxe master
```

**Comment by tehnick** [comment]
Done.

**Comment by starius** [comment]
Thanks!

**Pull request merged**